services/api-server/src/simcore_service_api_server/api/routes/studies_jobs.py

```diff
@@ -284,7 +276,6 @@ async def get_study_job_custom_metadata(
     "/{study_id}/jobs/{job_id}/metadata",
     include_in_schema=API_SERVER_DEV_FEATURES_ENABLED,
     status_code=status.HTTP_501_NOT_IMPLEMENTED,
-    response_description="Not implemented",
```

Comment:
I think the main reason for adding this originally was that it makes it clear from the swagger site that the endpoint is not yet implemented, which I find useful. Maybe it would make sense to check if it is also clear from the swagger view now that you remove this line.  🤔